presto-main/src/main/java/com/facebook/presto/server/HttpRemoteTask.java

```diff
@@ -454,6 +455,14 @@ private synchronized void scheduleUpdate()
 
         updateErrorTracker.startRequest();
 
+        // TODO: (billg) remove this logging or contribute it upstream
+        if (log.isDebugEnabled()) {
+            String size = "unknown";
+            if (request.getBodyGenerator() instanceof StaticBodyGenerator) {
+                size = Integer.toString(((StaticBodyGenerator) request.getBodyGenerator()).getBody().length);
```

Comment:
You don't need to use Integer.toString.

Just use %d in the String.format() for the bodySize